test-clients/src/main/java/com/hedera/services/yahcli/suites/ScheduleSuite.java

```diff
@@ -40,7 +40,8 @@ public List<HapiSpec> getSpecsInSuite() {
     }
 
     private HapiSpec doSchedule() {
-        var schedule = new HapiScheduleSign("0.0." + scheduleId);
+        // 0.0 is the default payer
```

Comment:
`0.0` isn't an account, right? I think `DEFAULT_SHARD` is enough and we can get rid of this comment